IDWiseModular_Example/example-module/Sources/ExampleModule/ExampleType.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  ExampleType.swift
+//  ExampleModule
+//
+//  Created by Mian Najeeb on 08/03/2023.
+//
+
+import Foundation
+import IDWise
+
+public class ExampleType {
+    
+    public static func initialiseIDWiseSDK() {
+        IDWise.initialize(clientKey: "CLIENT_KEY", onError: { _ in })
+    }
+}
```

Comment:
@najeeb-ios  add the start journey method and protocol methods here as well as you did in other demo projects.

Reply:
remove static method and add IDWise methods simply as you did in other projects and also add protocol IDWiseJourneyDelegatge methods for simple journey only

Reply:
Updated